NuGet.config

```diff
@@ -16,5 +16,15 @@
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
     <add key="dotnet5-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5-transport/nuget/v3/index.json" />
   </packageSources>
-  <disabledPackageSources />
+  <packageSourceMapping>
+    <packageSource key="dotnet-public">
+      <package pattern="*" />
```

Comment:
NIT: If you are only requiring these three package source mappings, it means that all of the feeds that you have under <!-- products --> are not needed and can be cleaned up.